src/mono/wasm/host/WebServerStartup.cs

```diff
@@ -82,7 +82,7 @@ static int GetNextRandomExcept(Range range, params int[] except)
         provider.Mappings[".cjs"] = "text/javascript";
         provider.Mappings[".mjs"] = "text/javascript";
 
-        foreach (string extn in new string[] { ".dll", ".pdb", ".dat", ".blat" })
+        foreach (string extn in new string[] { ".dll", ".pdb", ".dat", ".blat", ".webcil" })
```

Comment:
Please add it also to https://github.com/dotnet/runtime/blob/7131ef92b963b9fc1f5b42abe5eabb90da38ba57/src/mono/sample/wasm/simple-server/Program.cs#L157

And https://github.com/dotnet/runtime/blob/7131ef92b963b9fc1f5b42abe5eabb90da38ba57/src/mono/sample/wasm/Directory.Build.targets#L17

And to xharness web server 
https://github.com/dotnet/xharness/blob/main/src/Microsoft.DotNet.XHarness.CLI/Commands/WebServer.cs#L119